### ISSUE TITLE
Dark theme issues

### ISSUE THREAD

**Issue by msoupcoff** [sakaicontrib/attendance#94]
Certain colors look to be hardcoded and thus look out of place or fail color contrast test when switching to dark theme. Also reported in core jira project (https://sakaiproject.atlassian.net/browse/SAK-46981)

1. On **Attendance** tab:
![AttendanceTab](https://user-images.githubusercontent.com/1220785/155371099-1687b553-dca7-43b8-9db3-04c5724fca3d.png)

1. After selecting an item to take attendance:
![TakeAttendance](https://user-images.githubusercontent.com/1220785/155371670-7d13fe32-d23e-42e3-8c2b-2ea73ce87cfe.png)

1. After selecting an item to take attendance & selecting the **Edit attendance item** icon:
![TakeAttendanceEditItem](https://user-images.githubusercontent.com/1220785/155372090-29666041-74b5-43b2-915b-2c219b7e45e0.png)

1. After selecting an item to take attendance & selecting the **Print** icon:
![TakeAttendancePrint](https://user-images.githubusercontent.com/1220785/155372289-947a6317-d284-40ed-9e8c-723756f2acb4.png)

1. After selecting an item to take attendance & scroll over students (odd rows):
![TakeAttendanceRowScrollover](https://user-images.githubusercontent.com/1220785/155372472-a58d3157-c418-4634-8caf-fc3831d93f12.gif)

1.  After selecting an item to take attendance & selecting a student's name
![AttendanceSelectedStudent](https://user-images.githubusercontent.com/1220785/155372788-0110d535-cf51-4129-b481-bd0bedd3be83.png)

1. On **Attendance By Student** tab:
![AttendanceByStudent](https://user-images.githubusercontent.com/1220785/155373557-232c3243-f6f3-43e6-bcd7-a684051f89e1.png)

1. On **Grading** tab:
![AttendanceGrading](https://user-images.githubusercontent.com/1220785/155372615-7fdcf0f8-b998-4f94-aade-dd844d7d2c19.png)

1. On **Settings** tab:
![AttendanceSettings](https://user-images.githubusercontent.com/1220785/155372666-20fbdb2f-4d8b-40f9-bd13-1293a3fc1bf9.png)

**Comment by profmikegreene** [comment]
You can assign this to me if no one on your team isn't already working on it.

**Comment by davidpbauer** [comment]
We have made some fixes to the dark theme locally. I can see if Joe who made the changes can make a pull request. It would at least be a place to start if you have more pressing things.
